### PR TITLE
update docs for multi-realm and fix developers.signalfx links

### PR DESCRIPTION
**Pull request by mstumpfx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +34,28 @@ installing a more recent gem.  Building and installing signalfx from source will
...
+
+In order to send your data to the correct realm, you may need to configure your 
+endpoints. If no endpoints are set manually, this library uses the ``us0`` realm by default. 
+If you are not in this realm, you will need to explicitly set the
+endpoint config options below. To determine if you are in a different realm and need to
+explicitly set the endpoints, check your profile page in the SignalFx 
+web application. You will also need to specify an access token when requesting
+one of those clients. For the ingest client, you need to specify your
+organization access token (which can be obtained from the
+SignalFx organization you want to report data into). For the REST API,
```
Comment (barbara-sfx) [inline README.md]: I commented about the REST token on another PR.

File: README.md [hunk 2]
```diff
@@ -31,6 +34,28 @@ installing a more recent gem.  Building and installing signalfx from source will
...
+
+```ruby
+ingest_endpoint https://ingest.{REALM}.signalfx.com
+api_endpoint https://api.{REALM}.signalfx.com
+stream_endpoint https://stream.{REALM}.signalfx.com
+```
+
 ### API access token
 
 To use this library, you need a SignalFx API access token, which can be obtained from the SignalFx organization you want to report data into.
```
Comment (barbara-sfx) [inline README.md]: Terminology needs to be consistent (even though it's confusing). Do you mean org access token? (supports ingest and all APIs except member management).

File: README.md [hunk 3]
```diff
@@ -4,6 +4,9 @@ This is a programmatic interface in Ruby for SignalFx's metadata and
 ingest APIs. It is meant to provide a base for communicating with
 SignalFx APIs that can be easily leveraged by scripts and applications
 to interact with SignalFx or report metric and event data to SignalFx.
+You will need a SignalFx account and organization API token to use them.
```
Comment (barbara-sfx) [inline README.md]: It seems weird that we are telling them how to get an account. We don't do that anywhere else. Can this be consistent with other docs?

**Review by barbara-sfx — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -4,6 +4,7 @@ This is a programmatic interface in Ruby for SignalFx's metadata and
 ingest APIs. It is meant to provide a base for communicating with
 SignalFx APIs that can be easily leveraged by scripts and applications
 to interact with SignalFx or report metric and event data to SignalFx.
+You will need a SignalFx account and an access token to use them.
```
Comment (barbara-sfx) [inline README.md]: I don't get why you are starting to add this to the docs. It's pretty much true of every page in every doc we produce. Did someone ask you to add this?

Comment (mstumpfx) [reply README.md]: just copy pasting from other libraries. i can remove

**Pull request merged**